### PR TITLE
Added Login and Signup Page

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Added 
- Login Page
- SignUp Page
- Remember Me while Login Feature 

Here is the Preview Link :
https://deploy-preview-2--distracted-wilson-c25ae0.netlify.app/authentication/login.html

### CODE REVIEW

File: authentication/auth.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+}
+.forgot-pswrd-link{
+    color: var(--primary-color);
+    text-decoration: none;
+}
+.login-btn{
+    color: white;
+    font-weight: bolder;
+    border: none;
+    font-size: 17px;
```
Comment (ozairkhan2810) [inline authentication/auth.css]: use rem

File: authentication/auth.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    padding: 10px;
+    border-radius: 5px;
+    background-color: var(--primary-color);
+    cursor: pointer;
+}
+.login-btn:hover{
+    background-color: #5b5bf8;
```
Comment (ozairkhan2810) [inline authentication/auth.css:63]: use variable name

File: authentication/auth.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+}
+.have-account-link{
+    text-align: center;
+    text-decoration: none;
+    color: var(--primary-color);
+}
+/* SignUp page */
+.user-name{
+    display: flex;
+    gap: 15px;
```
Comment (ozairkhan2810) [inline authentication/auth.css]: use rem

**Review by ozairkhan2810 — APPROVED** [review]
refactor the changes and merge

**Review by shaksham08 — APPROVED** [review]
Looks good just follow the comments what ozairkhan has added

File: authentication/auth.css (hunk 1 above)
Comment (atharvabhanage02) [reply authentication/auth.css]: Done !

**Pull request merged**